wheel/Cargo.toml

```diff
@@ -37,6 +37,13 @@ tonic = "0.6.2"
 tracing = "0.1"
 tracing-subscriber = "0.3"
 
+prometheus = "0.13.0"
+lazy_static = "^1.4"
+chrono = { version = "0.4", features = ["serde"] }
+hyper = { version = "^0.14", features = ["server", "http1", "tcp"] }
+http = "0.2.6"
+log = "0.4.16"
```

Comment:
We don't need `log` crate. In RunKV we use `tracing` to record event and span logs. Please remove the dependency.